nexus-sdk/build.gradle

```diff
@@ -53,7 +53,7 @@ import com.vanniktech.maven.publish.SonatypeHost
 mavenPublishing {
     publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL)
 
-    signAllPublications()
```

Comment:
Note: need to remove this before merging, this line just makes local development impossible